libs/types/src/lib/dataset.ts

```diff
@@ -64,3 +67,13 @@ interface Sample {
   mediaType?: string[];
   description?: LocalizedStrings;
 }
+export interface Reference {
+  referenceType: { code: string };
+  source: { uri: string };
+}
+
+export interface DatasetSerie {
```

Comment:
Er "series" i både entall og flertall
```suggestion
export interface DatasetSeries {
```